### PR TITLE
Add nsCloudConfigManager to the public API

### PR DESCRIPTION
**Pull request by TsvetanMilanov**
Add `nsCloudConfigManager` to the public API. It should provide methods for managing the cloud configuration and for listing all cloud services domain names.

This PR also removes the unused cloud services.

### CODE REVIEW

File: lib/cloud-config-manager.ts [hunk 1]
```diff
@@ -1,10 +1,10 @@
 import * as path from "path";
 
-export class ServerConfigManager implements IServerConfigManager {
+export class ServerConfigManager implements ICloudConfigManager {
```
Comment (tsvetie) [inline lib/cloud-config-manager.ts]: Shouldn't you rename this to "CloudConfigManager"?

File: lib/cloud-config-manager.ts [hunk 2]
```diff
@@ -39,6 +39,37 @@ export class ServerConfigManager implements IServerConfigManager {
...
+		if (_.has(serviceConfig, valueName)) {
+			return serviceConfig[valueName];
+		}
+
+		return <string>config[valueName];
+	}
+
+	public getCloudServicesDomainNames(): string[] {
+		const cloudServices = this.getCurrentConfigData().cloudServices;
+		return _(cloudServices)
```
Comment (tsvetie) [inline lib/cloud-config-manager.ts]: Empty line above return?

**Comment by TsvetanMilanov** [comment]
No need to test it. It works on my machine 😄

File: lib/cloud-config-manager.ts (hunk 2 above)
Comment (TsvetanMilanov) [reply lib/cloud-config-manager.ts]: Done.

File: lib/cloud-config-manager.ts (hunk 1 above)
Comment (TsvetanMilanov) [reply lib/cloud-config-manager.ts]: 🎣

**Pull request merged**